priv/schema/rabbitmq_web_mqtt.schema

```diff
@@ -0,0 +1,44 @@
+{mapping, "web_mqtt.num_acceptors.tcp", "rabbitmq_web_mqtt.num_tcp_acceptors", 
+    [{datatype, integer}]}.
+{mapping, "web_mqtt.num_acceptors.ssl", "rabbitmq_web_mqtt.num_ssl_acceptors", 
+    [{datatype, integer}]}.
+
+{mapping, "web_mqtt.tcp_config.port", "rabbitmq_web_mqtt.tcp_config.port",
```

Comment:
We have "_opts" in other places and they don't seem to be too bad to me. "_config" is something that is clearly extraneous: every single setting in a config file is… well, some kind of configuration ;)